test/addr_book_suite.go

```diff
@@ -213,27 +213,27 @@ func testAddressesExpire(m pstore.AddrBook) func(t *testing.T) {
 		testHas(t, []ma.Multiaddr{ma24, ma25}, m.Addrs(id2))
 
 		m.SetAddr(id1, ma11, time.Millisecond)
-		<-time.After(time.Millisecond * 2)
+		<-time.After(time.Millisecond * 5)
```

Comment:
@raulk I thought this PR was just a structural refactor, why would these timeouts need increasing?

Reply:
This test is time-dependent and the wait times were too tight. It was already failing prior to this PR: 
https://travis-ci.org/libp2p/go-libp2p-peerstore/builds/421327878.

Since @magik6k mentioned it above, I figured I’d kill the red herring by push a fix.